zerver/tests/test_i18n.py

```diff
@@ -163,6 +163,12 @@ def test_get_browser_language_code(self) -> None:
         req.META["HTTP_ACCEPT_LANGUAGE"] = "*"
         self.assertIsNone(get_browser_language_code(req))
 
+        # Case when language with percent_translated less than
+        # 5 has higher weight.
+        req = HostRequestMock()
+        req.META["HTTP_ACCEPT_LANGUAGE"] = "no;q=0.9,de;q=0.8"
```

Comment:
We may regret this if `no` ever gets a complete translation. But maybe it's a pain in test setup to make one up...